changes.md

```diff
@@ -1,3 +1,22 @@
+# PostgreSQL installer release 18.1-1 (2025-11-13)
+## Changes 🛠️
+- **Dependencies (MacOS & Windows)**
+    - Update `pgAdmin4` to version `9.9`
```

Comment:
Not a strong opinion, but pgAdmin is not a dependency, so should it be listed outside of Dependencies?